main.py

```diff
@@ -218,14 +229,663 @@ def queries(data,graph):
+    for answers in Q2:
+        for _key,item in answers.items():
+            for _key,item2 in item.items():
+                filee.write('{:<30}'.format(str(item2)))
+            filee.write("\n")
+    filee.write("\n\n\n")
+
+    filee.write("3. How many actors have acted in same movies with ’Edward Norton’?\n")
+    for answer in Q3:
+        for _key,item in answer.items():
+            filee.write('{:<30}'.format(str(item)))
+        filee.write("\n")
+    filee.write("\n\n\n")
+
+    filee.write("4. Which collectors collected all movies in which ’Edward Norton’ acts?\n")
+    filee.write("Userid                          Fullname \n")
+    for answer in Q4:
+        for _key,item in answer.items():
+            filee.write('{:<30}'.format(str(item)))
+        filee.write("\n")
+    filee.write("\n\n\n")
+
+    filee.write("5. List 10 collectors ( userid and fullname ) who collect ’The Shawshank Redemption’.\n")
+    filee.write("Fullname                     Userid \n")
+    for answer in Q5:
+        for _key,item in answer.items():
+            filee.write('{:<30}'.format(str(item)))
+        filee.write("\n")
+    filee.write("\n\n\n")
+
+    filee.write("6. List all userids and fullnames of xi’s which satisfy Degree(1001, xi) ≤ 3\n")
+    filee.write("Fullname\n")
+    for answers in Q6:
+        for _key,item in answers.items():
+            for collector in item:
+                filee.write('{:<30}'.format(str(collector)))
+                filee.write("\n")
+
+###################################### CASSANDRA ##################################################
+def cassandra(data,hostname):
+    try:
+        # Connect to server
+        cluster = Cluster([hostname])
+        session = cluster.connect()
+    # If server is not connected :
+    except Exception:
+        print ("Unable to reach server.")
+        sys.exit()
+    #Create a KEYSPACE and USE it
+    a=session.execute("""CREATE KEYSPACE IF NOT EXISTS mykeyspace WITH replication = { 'class': 'SimpleStrategy', 'replication_factor': '1' }""")
+    session.execute('USE mykeyspace')
+    #Clear tables for executing again
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.movies""")
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.actors""")
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.directors""")
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.collectors""")
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.ACTED_IN""")
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.ACTED_IN_q3""")
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.DIRECTED""")
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.COLLECTS""")
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.FOLLOWS""")
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.FOLLOWS_q6""")
+    session.execute("""DROP TABLE IF EXISTS mykeyspace.actors_directors""")
+
+    #Create tables
+    session.execute(""" CREATE TABLE IF NOT EXISTS movies(

```

Comment:
Tabi o zaman tüm keyspace'i varsa bile sildiğimiz için buralarda da direk
CREATE TABLE Movie
diyebiliriz